components/SearchBar/index.tsx

```diff
@@ -12,6 +13,9 @@ export default function SearchBar({
 }: SearchBarProps) {
   return (
     <SearchBarContainer>
+      <IconWrapper>
+        <Icon type="search" />
+      </IconWrapper>
```

Comment:
nit: we might be able to add the icon directly in the input styles, [here's](https://www.w3schools.com/css/css_form.asp) a reference of using the `background` attribute to load an svg. Here's more info on the [background css attribute](https://www.w3schools.com/css/css3_backgrounds.asp).

Here's what ChatGPT said about the pros/cons of these 2 approaches: 
<img width="607" alt="Screen Shot 2024-11-24 at 3 08 06 PM" src="https://github.com/user-attachments/assets/316506b2-2018-4597-916a-673cc3b6f684">

* if we want to make the search icon interactive, perhaps we can leave it as it now! but just wanted to note that there is a different way to add icons too!